scripts/install-monero-linux.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
-if [[ -d "./monero-x86_64-linux-gnu-v0.17.3.2" ]]; then 
-    echo "monero-x86_64-linux-gnu-v0.17.3.2 already installed"
+arch=linux64
+
+if [[ -d "monero-bin" ]]; then
+    echo "$(dirname $(realpath monero-bin)) already installed"
     exit 0
 fi
```

Comment:
`monero-bin` is a symbol link.  All scripts now reference it instead of the arch/release dependent name.  In the test above, both the symbolic link and the directory it refers to must exist in order to take the early exit path (you can delete either one to update).